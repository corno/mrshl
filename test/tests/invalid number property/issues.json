[
    [ "value 'true' is not a number", "error", 2, 10, 2, 14 ]
]