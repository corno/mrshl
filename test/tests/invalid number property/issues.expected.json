[
	[
		"validation",
		"error",
		"value 'true' is not a number",
		2,
		10,
		2,
		14
	]
]